shared/src/main/java/com/wedriveu/shared/util/Constants.java

```diff
@@ -249,6 +249,15 @@
              *  The routing key used by backoffice to receive a vehicle counter.
              */
             String ROUTING_KEY_ANALYTICS_RESPONSE_VEHICLE_LIST = "analytics.response.vehicles";
+            /**
+             *  The routing key used by booking service to receive a booking list request.
+             */
+            String ROUTING_KEY_BOOKING_REQUEST_BOOKING_LIST = "booking.request.bydate";
```

Comment:
Please, didn't you notice the `RabbitMq.RoutingKey` interface in this file? There goes all the Routing Keys! And please also remove the `ROUTING_KEY` prefix. Please make this change also for analytics routing keys.

Reply:
They are already in the routing key interface, i'll remove the routing key prefix